pennylane/vqe/vqe.py

```diff
@@ -123,8 +154,14 @@ def __init__(self, coeffs, observables, simplify=False, id=None, do_queue=True):
 
         self.return_type = None
 
+        # attribute to store indices used to form groups of
+        # commuting observables, since recomputation is costly
+        self._grouping_indices = None
+
         if simplify:
             self.simplify()
+        if compute_groupings:
+            self._grouping_indices = qml.transforms.invisible(_compute_grouping_indices)(self.ops)
```

Comment:
Needs to be invisible because Pauli Operations are created and they can be misinterpreted as gates in the queue!!

Reply:
I know, this is something we have to address next quarter!